api/src/team/infrastructure/serializers/jsonapi/organization-invitation.serializer.js

```diff
@@ -17,5 +22,5 @@ const deserializeForCreateOrganizationInvitationAndSendEmail = function (payload
     };
   });
 };
-
-export { deserializeForCreateOrganizationInvitationAndSendEmail, serialize };
+const serializer = { serialize, deserializeForCreateOrganizationInvitationAndSendEmail };
+export { deserializeForCreateOrganizationInvitationAndSendEmail, serialize, serializer };
```

Comment:
```suggestion
export const organizationInvitationSerializer = { deserializeForCreateOrganizationInvitationAndSendEmail, serialize }
```

ce qui permettra d'enlever les `import as ...` dans les controller 😄